platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/worker_api.adoc

```diff
@@ -13,50 +13,53 @@
 // limitations under the License.
 
 [[tasks_parallel_worker]]
-= Developing Parallel Tasks using the Worker API
+= Developing Parallel Tasks
+
+Gradle provides an API that can split tasks into sections that can be executed in parallel.
+
+image::writing-tasks-5.png[]
```

Comment:
❗ I don't think this diagram is clear.  I think the difference in time is not immediately obvious, and instead of repeating "Task 7" it should say "Task 7" and also "Task 7 - Work 1", "Task 7 - Work 2" etc.